### PR TITLE
Mockup corporate - task 3 JS

### PR DESCRIPTION
**Pull request by maycatsby**
https://maycatsby.github.io/ui-internship-lwo2/HTML/mockup-corporate/

### CODE REVIEW

File: HTML/mockup-corporate/js/task3.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+const validArr = [false, false, false];
+
+inputNameArr.forEach((el, ind) => {
+  el.addEventListener('input', () => {
+    let validValue = validator(inputNameArr[ind], patternArr[ind]);
+    validArr[ind] = validValue;
+
+    elements.btn.classList.remove('btn--disabled');
```
Comment (tkachukivan) [inline HTML/mockup-corporate/js/task3.js]: there is disabled attribute better to use one, styles for disabled button make this button almost invisible, what is not good ui

File: HTML/mockup-corporate/js/task3.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    return false;
+  } else {
+    return true;
+  }
+};
+
+const validArr = [false, false, false];
+
+inputNameArr.forEach((el, ind) => {
+  el.addEventListener('input', () => {
```
Comment (tkachukivan) [inline HTML/mockup-corporate/js/task3.js]: maybe it is possible to have just one listener using [event delegation](https://javascript.info/event-delegation), which is generally very useful technic, which based on [bubbling and capturing of events](https://javascript.info/bubbling-and-capturing)

**Review by tkachukivan — COMMENTED** [review]
take a look at comments

**Review by tkachukivan — APPROVED** [review]

File: HTML/mockup-corporate/sass/components/_input.scss [hunk 3]
```diff
@@ -4,12 +4,13 @@
...
     border-radius: 4px;
     font-size: 1.4rem;
+    color: $color-mode-white;
     background: none;
     background-color: #f6f5f311;
 
     &:focus {
         outline: none;
-        border-color: $color-main-light;
+        // border-color: rgb(127, 212, 127);
```
Comment (tsvetkovv) [inline HTML/mockup-corporate/sass/components/_input.scss:13]: commented code?

**Review by tsvetkovv — APPROVED** [review]
good

**Pull request merged**